### PR TITLE
Fix media queries for Outlook on ios

### LINKED ISSUE #193: Style Inliner - Responsive CSS not working as expected on Outlook iPhone
We noticed that when an Email is sent through Adobe Campaign using the HTML source generated from the Style Inliner servlet as is, the contents of the Email are not responsive on Outlook iPhone.

Where as when the CSS from the Style Inliner source is well formatted (pretty-printed) using a CSS formatter the Email renders correctly on Mobile.

This issue is very specific to Outlook on iPhone, this is not an issue on Outlook Desktop App on Windows.

#### Sample Code that illustrates the problem from Style Inliner
`<style type="text/css">@media (max-width: 639px) {.root { width: 100% !important; }}@media (max-width: 639px) {.sample-email-root > center > table { width: 100% !important; }}.sample-email-root > center table span {font-family: Arial;}</style>`

### Not an issue when CSS is formatted as below
```
<style type="text/css">
 @media (max-width: 639px) {
     .root {
         width: 100% !important;
    }
}
 @media (max-width: 639px) {
     .sample-email-root > center > table {
         width: 100% !important;
    }
}
 .sample-email-root > center table span {
     font-family: Arial;
}
</style>
```

[Sample Code - Issue with Core Email Text Component.zip](https://github.com/adobe/aem-core-email-components/files/9242482/Sample.Code.-.Issue.with.Core.Email.Text.Component.zip)

### PR DESCRIPTION
**Pull request by bpauli**
- add extra space between selector and properties
- adapt unit tests

fixes #193 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: bundles/core/src/main/java/com/adobe/cq/email/core/components/util/StyleTokenFactory.java [hunk 1]
```diff
@@ -133,11 +134,17 @@ public static void addProperties(StyleToken styleToken, String properties) {
      * @param styleToken the {@link StyleToken}
      * @return a valid CSS representation
      */
+    @Nullable
     public static String toCss(StyleToken styleToken) {
-        if (Objects.isNull(styleToken)) {
-            return null;
+        String css = null;
+        if (Objects.nonNull(styleToken)) {
+            if (styleToken.isMediaQuery()) {
```
Comment (Buuhuu) [inline bundles/core/src/main/java/com/adobe/cq/email/core/components/util/StyleTokenFactory.java:141]: Why do we make a difference here between media query and other selectors? Could we use the same pattern (with the spaces) for all?

Comment (bpauli) [reply bundles/core/src/main/java/com/adobe/cq/email/core/components/util/StyleTokenFactory.java]: @Buuhuu as discussed offline I updated the PR to unify the behaviour for all css selectors and adapt the formatting to the behaviour in the mjml project.

**Review by Buuhuu — APPROVED** [review]

File: bundles/core/src/main/java/com/adobe/cq/email/core/components/util/StyleTokenFactory.java [hunk 2]
```diff
@@ -133,11 +134,13 @@ public static void addProperties(StyleToken styleToken, String properties) {
      * @param styleToken the {@link StyleToken}
      * @return a valid CSS representation
      */
+    @Nullable
     public static String toCss(StyleToken styleToken) {
-        if (Objects.isNull(styleToken)) {
-            return null;
+        String css = null;
+        if (Objects.nonNull(styleToken)) {
+            css = styleToken.getSelector() + " { " + getAllProperties(styleToken) + " }\n";
```
Comment (edoardo-goracci) [inline bundles/core/src/main/java/com/adobe/cq/email/core/components/util/StyleTokenFactory.java:141]: You can probably replace the "\n" character at line 141 with another space

**Review by edoardo-goracci — APPROVED** [review]
Everything looks fine to me, I just added a comment for StyleTokenFactory

File: bundles/core/src/main/java/com/adobe/cq/email/core/components/util/StyleTokenFactory.java (hunk 2 above)
Comment (bpauli) [reply bundles/core/src/main/java/com/adobe/cq/email/core/components/util/StyleTokenFactory.java:141]: I will keep the new line for now for each selector like this is done by the MJML framework. If we see issues with the JSON exchange between AEM and ACC we should probably remove it.

**Pull request merged**